content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+
+
+Accounts
+========
+
+**Enable multi factor authentication in Gmail.**
+
+Use a hardware key (Yubikey) or mobile, as long as there is 2FA involved
+
+Enforcement: Go to the list of accounts in the organization and see which one has 2FA.  If the account does not have 2FA enabled, contact the account holder.
+
+**Enable multi factor authentication in Github**
+
+Idea: Enable this in Github settings - can use mobile device, hardware key, downloadable codes, etc.
+
+Enforcement: The Google organization owner/admin can see who has MFA.  If MFA is not enabled, contact the account owner to re-enable.
+
+**Enable multi-factor authentication and valid back up** in other services used by SFTPPlus.
+
+
+Email
+=====
+
+
+Enable S/MIME in Emails
+-----------------------
+
+There are free email certificates available.  
+
+A list of S/MIME email certificates that you can use is at http://kb.mozillazine.org/Getting_an_SMIME_certificate
+
+You can use the Comodo email certificate. To obtain a free email certificate from Comodo, go to https://www.instantssl.com/ssl-certificate-products/free-email-certificate.html and sign up to recieve the certificate in your Pro:Atria email. To obtain the Comodo cert in PKCS#12 format, collect the cert in the confirmation email, open Mozilla and go to Advanced > Certificates > View Certificates and backup the cert to obtain the PKCS#12 format.
+
+If you use and recommend another provider please add details in this section.
+
+Once obtained, you can add the digital certificate to your email provider of choice.
+
+
+I want to use PGP/MIME?
+-----------------------
+
+You cannot use both S/MIME and PGP/MIME at the same time.
+PGP/MIME really only makes sense in certain situations.  The general use one is S/MIME.
+
+
+Other email tips
+----------------
+
+Add the Pro:Atria Email Signature Template in your emails.
+
+Set your email settings to prevent HTML/images loading and to block remote content.  
+
+Utilize a blacklist of known spam senders.
+
+
+Use of group emails
+-------------------
+
+Do not add team members that are still on trial to the group until such addition is verified by the team lead.
+
+Do not use team group emails for non SFTPPlus purposes like personal and industry newsletters, announcements, and so on.
+
+There may be some cases where we need to use the group email for related services. When using team group emails for SFTPPlus testing or to open SFTPPlus accounts please notify the group email first that the testing / account is legitimate.  That way, when a test is sent to a team account it is not immediately assumed that it is legitimate. 
+
+
+Phishing and Social Engineering
+===============================
+
+If a phishing attack is encountered or you suspect a phishing attack is encountered notify the team.  
+
+Phishing is also utilized as part of active information gathering while enumerating the target and also points to some form of information gathering/enumerating is taking place.
```

Comment:
The second part doesn't sound right to me… Perhaps _also points to some form of information gathering (eg. enumarating) taking place_?